tests/functional/workloads/cnv/test_cnv_node_replacement.py

```
@@ -86,69 +86,95 @@ def test_vms_with_node_replacement(
-        # Find node where VM is running
+        logger.test_step("Set VM states and perform node replacement")
         node_name = self.vm_obj_on_replacing_node.get_vmi_instance().node()
+        logger.info(f"Target node for replacement: '{node_name}'")
 
-        # Stop VM if its not live migratable to avoid drain stuck issue.
         for vm_rwo in all_vms:
             if (
                 vm_rwo != self.vm_obj_on_replacing_node
                 and vm_rwo.get_vmi_instance().node() == node_name
             ):
                 if vm_rwo.pvc_access_mode == "ReadWriteOnce" and vm_rwo.ready():
```

Ruff (0.15.21)

[warning] Use a single `if` statement instead of nested `if` statements

(SIM102)